app/Filament/Resources/StandResource.php

```diff
@@ -0,0 +1,247 @@
+                                fn () => Airfield::all()
+                                    ->sortBy('code', SORT_NATURAL)
+                                    ->mapWithKeys(fn (Airfield $airfield) => [$airfield->id => $airfield->code])
+                            )
+                            ->reactive()
+                            ->afterStateUpdated(function (Closure $get, Closure $set) {
+                                $terminalId = $get('terminal_id');
+                                if ($terminalId && Terminal::find($terminalId)->airfield_id === $get('airfield_id')) {
+                                    return;
+                                }
+
+                                $set('terminal_id', null);
+                            })
+                            ->preload()
+                            ->searchable(!App::runningUnitTests())
+                            ->disabled(fn (Page $livewire) => !$livewire instanceof CreateRecord)
+                            ->dehydrated(fn (Page $livewire) => $livewire instanceof CreateRecord)
+                            ->required(),
+                        Select::make('terminal_id')
+                            ->label(__('form.stands.terminal.label'))
+                            ->helperText(__('form.stands.terminal.helper'))
+                            ->hintIcon('heroicon-o-folder')
+                            ->options(
+                                fn (Closure $get) => Terminal::where('airfield_id', $get('airfield_id'))
+                                ->get()
+                                ->mapWithKeys(
+                                    fn (Terminal $terminal) => [$terminal->id => $terminal->description]
+                                )
+                            )
+                            ->disabled(
+                                fn (Page $livewire, Closure $get) => !$livewire instanceof CreateRecord ||
+                                    !Terminal::where('airfield_id', $get('airfield_id'))->exists()
+                            )
+                            ->dehydrated(
+                                fn (Page $livewire, Closure $get) => !$livewire instanceof CreateRecord ||
+                                    !Terminal::where('airfield_id', $get('airfield_id'))->exists()
+                            ),
+                        TextInput::make('identifier')
+                            ->label(__('form.stands.identifier.label'))
+                            ->maxLength(255)
+                            ->helperText(__('form.stands.identifier.helper'))
+                            ->required()
+                            ->rule(
+                                fn (Closure $get, ?Model $record) => $get('airfield_id') ? new StandIdentifierMustBeUniqueAtAirfield(
+                                    Airfield::findOrFail($get('airfield_id')),
+                                    $record
+                                ) : function () {
```

Comment:
What's occurring here with the empty closure?

Reply:
Changed this to use a conditional: e54e294b